packages/jss/.size-snapshot.json

```diff
@@ -10,21 +10,21 @@
     "gzipped": 5827
   },
   "dist/jss.cjs.js": {
-    "bundled": 48623,
-    "minified": 20945,
-    "gzipped": 5971
+    "bundled": 54298,
+    "minified": 24068,
+    "gzipped": 6908
```

Comment:
Why is there such a significant increase in size?

Reply:
Ooops, and here's the problem. node-globals unconditionally polyfills everything. Even `process` object which is used by us for NODE_ENV.

Reply:
Added PR
https://github.com/calvinmetcalf/rollup-plugin-node-globals/pull/16

Reply:
I mean we only need to shim `process.env.NODE_ENV`, correct? 
And this should always be replaced by the user.

Reply:
We only need to shim `global` but `process.env.NODE_ENV` should be replaced on user side.

Reply:
shit I didn't see the size increase

Reply:
Seems like umd bundles size is also increased since `nodeGlobals` is used before `replace`.

Reply:
The simplest work around is own ponyfill which we could reuse in modules.

Reply:
are we using anything from `global` other than `process.env.NODE_ENV` or `process.env.VERSION`?

Reply:
`global` is used to create unique identifiers and nothing more. `process.env.VERSION` is replaced statically on our side. `process.env.NODE_ENV` should be replaced statically on user side.